src/lightning_lite/lite.py

```diff
@@ -496,3 +524,11 @@ def _validate_setup_dataloaders(dataloaders: Sequence[DataLoader]) -> None:
 
         if any(not isinstance(dl, DataLoader) for dl in dataloaders):
             raise TypeError("Only PyTorch DataLoader are currently supported in `setup_dataloaders`.")
+
+
+def _is_using_cli() -> bool:
+    return bool(int(os.environ.get("LT_CLI_USED", "0")))
```

Comment:
```suggestion
    return bool(os.environ.get("LT_CLI_USED", False))
```

Reply:
No, this is problematic. 
`bool("0") is True`